src/zabapgit_dot_abapgit.prog.abap

```diff
@@ -12,11 +12,18 @@ CLASS lcl_dot_abapgit DEFINITION FINAL FRIENDS ltcl_dot_abapgit.
                  full   TYPE string VALUE 'FULL',
                END OF c_folder_logic.
 
-    TYPES: BEGIN OF ty_dot_abapgit,
+    TYPES: BEGIN OF ty_requirement,
+             component   TYPE dlvunit,
+             min_release TYPE saprelease,
+             min_patch TYPE sappatchlv,
+           END OF ty_requirement,
+           ty_requirement_tab TYPE STANDARD TABLE OF ty_requirement WITH DEFAULT KEY,
```

Comment:
Is it possible to rename to `ty_requirement_tt`? I see this is the pattern used in https://github.com/larshp/abapGit/blob/master/src/zabapgit_definitions.prog.abap